portal-web/test/functional/com/liferay/portalweb/macros/SearchExperiences.macro

```diff
@@ -0,0 +1,238 @@
+definition {
+
+	macro addBlueprint {
+		Navigator.gotoNavItem(navItem = "Blueprints");
+
+		Click(locator1 = "Button#PLUS");
+
+		PortletEntry.inputTitle(title = "${name}");
+
+		if (isSet(description)) {
+			PortletEntry.inputDescription(description = "${description}");
+		}
+
+		if (isSet(clauseOption)) {
+			LexiconCard.clickCard(card = "Custom Clauses");
+		}
+		else {
+			LexiconCard.clickCard(card = "Liferay Default Clauses");
+		}
+
+		Button.click(button = "Create");
+
+		// Workaround for LPS-133591
+
+		Refresh();
+	}
+
+	macro addElements {
```

Comment:
I'd rather name this `addElement` to keep this consistent with our other macros that add a new asset entry and clarify situations where we add the existing element to a blueprint. I'm flexible on this one though.

Reply:
I was debating with myself on this because using `create` does make our macro naming less consistent. however, in this case it seemed worth it to opt for clarity at the cost of consistency because even someone who is familiar with blueprints would read the phrase "add element" in our tests and might not know what it is doing without having to read the code in the macro itself. but yeah if didn't affect anything practically, i would go with the consistent option.